invenio_communities/templates/invenio_communities/portalbox_upload.html

```diff
@@ -20,7 +20,7 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 #}
-<a href="TODO url for upload with community id" class="btn btn-primary btn-lg signup pull-right">Upload</a>
+<a href="{{ community.upload_url }}" class="btn btn-primary btn-lg signup pull-right">Upload</a>
```

Comment:
If we don't list Invenio-Deposit in setup.py then we should at least try/catch the build url error and return None and guard this section with `{% if community.upload_url %}`.